### PR TITLE
Refactor the Wine Engine Implementation Sourcecode

### PR DESCRIPTION
**Pull request by madoar**
This PR refactors the Wine engine implementation with some more modern Javascript operations

### CODE REVIEW

File: Engines/Wine/Engine/Implementation/script.js [hunk 1]
```diff
@@ -439,53 +468,63 @@ class WineEngine {
...
+            const IOUtils = Java.type("org.apache.commons.io.IOUtils");
+            return IOUtils.toString(process.getInputStream());
         } else {
             return "";
         }
     }
 
     changeVersion(containerName) {
-        var wizard = SetupWizard(InstallationType.ENGINES, "Change " + containerName + " container wine version", java.util.Optional.empty());
+        const wizardTitle = `Change ${containerName} container wine version`;
```
Comment (plata) [inline Engines/Wine/Engine/Implementation/script.js]: Should we have a `tr` here?

Comment (madoar) [reply Engines/Wine/Engine/Implementation/script.js]: Good idea! (Done)

File: Engines/Wine/Engine/Implementation/script.js [hunk 2]
```diff
@@ -483,7 +483,7 @@ class WineEngine {
     }
 
     changeVersion(containerName) {
-        const wizardTitle = `Change ${containerName} container wine version`;
+        const wizardTitle = tr("Change {0} container wine version", containerName);
```
Comment (plata) [inline Engines/Wine/Engine/Implementation/script.js]: While you're on it anyways: it should be "Wine".

Comment (madoar) [reply Engines/Wine/Engine/Implementation/script.js]: Done

**Pull request merged**